packages/functions/src/jobs/CronJobs.ts

```diff
@@ -0,0 +1,69 @@
+import { AppointmentStatus } from "common/src/types";
+import * as functions from "firebase-functions";
+import { getAppointmentsByStatus } from "../dal/AppointmentDataAccessLayer";
+import { getDonors } from "../dal/DonorDataAccessLayer";
+import { getAppointmentNotificationData } from "../notifications/AppointmentNotificationData";
+import {
+  NotificationToDonor,
+  sendEmailToDonor,
+} from "../notifications/NotificationSender";
+
+export const ConfirmationReminderOnSameDay = functions.pubsub
+  .schedule("every 1 hours")
+  .onRun(async () => {
+    const topOfThisHourInMillis = new Date().setMinutes(0, 0, 0);
+    const oneHourInMillis = 1000 * 60 * 60;
+    // Offset by 1 ms, to avoid catching XX:00 twice
+    const topOfPreviousHourInMillis =
+      topOfThisHourInMillis - oneHourInMillis + 1;
```

Comment:
So let's say the time now is 9:00.

Are you going to send pushes for appointments between 8:00:00.001 to 9:00?

In case this is correct I suggest adding a comment to make this clearer

Reply:
How would you suggest clarifying line #16?

Reply:
My comment wasn't about line 16 but was referring to these calculations.

I think something like this could be useful:

`A job triggered at 09:00 will send emails to all appointments with donationStartTime between  8:00:00.001 to 9:00`